app/src/main/res/xml/video_file_path.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<paths>
+    <external-files-path name="pictures" path="Movies/" />
```

Comment:
@tuancoltech Is "pictures" the correct name to use here?

Reply:
@jo-elimu It's corrected to `movies` now.